docs/configuration/overview/overview.rst

```diff
@@ -45,9 +45,11 @@ flags_path
   *(optional, string)* The file system path to search for :ref:`startup flag files
   <operations_file_system_flags>`.
 
-statsd_local_udp_port
-  *(optional, integer)* The UDP port of a locally running statsd compliant listener. If specified,
-  :ref:`statistics <arch_overview_statistics>` will be flushed to this port.
+statsd_udp_ip_address
```

Comment:
wondering if we just should throw to avoid confusion when we checked whether one or another is set.
https://github.com/lyft/envoy/pull/1019/files#diff-3eda0ff865e056f19775ad626a5c2920R321